website/content/Installation/_index.md

```diff
@@ -14,7 +14,7 @@ chapter = false
 $ curl https://raw.githubusercontent.com/databrickslabs/databricks-terraform/master/godownloader-databricks-provider.sh | bash -s -- -b $HOME/.terraform.d/plugins
 ```
 
-* The command should have moved the binary into your `~/.terraform.d/plugins` folder.
+* The command should have moved the binary into your `~/.terraform.d/plugins` folder.BRe
```

Comment:
@stuartleeks Umm not sure if the .BRe is a typo or not.

Reply:
oops! fixed